fast-af/fast-checkout-woocommerce
# Resolves PINT-666 - Add "Become a Seller" CTA in more places

# Description

Update the Fast settings page templates to include the "Become a Seller" CTA in more places when there is no Fast App ID.

# Testing instructions

1. Login to the staging environmet.
2. Go the Fast settings page.
3. Remove the App ID, but copy and paste it somewhere first because you will need to add it back when you are finished test.
4. Click on each tab and verify that the "Become a Seller" CTA is displayed under each tab.
5. Add the App ID back under the App Info tab.

# Checks
